pipeline/metadata/beam_metadata.py

```diff
@@ -2,9 +2,10 @@
 
 from __future__ import absolute_import
 
-from typing import Tuple, Dict, List, Iterator
+from copy import deepcopy
+from typing import Tuple, Dict, List, Iterator, Union, Iterable
 
-from pipeline.metadata.flatten import Row
+from pipeline.metadata.schema import BigqueryRow, SatelliteRow, IpMetadata, SatelliteAnswer, SatelliteAnswerMetadata, add_metadata_to_row, add_tags_to_answer
```

Comment:
Is the PR ready for review?

I don't see `add_metadata_to_row` anymore, for instance.